Server/DeviceManagement/DeviceManagement.Api/Dto/ErrorDto.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ModularHouse.Server.DeviceManagement.Api.Dto;
+
+public class ErrorDto
+{
+    public string ErrorMessage { get; set; }
```

Comment:
Add field `string StackTrace`, that shows stack trace of handled exception. Must be shown only in Development environment